api/tests/integration/domain/services/certification-candidates-ods-service/certification-candidates-ods-service_test.js

```diff
@@ -419,9 +421,10 @@ describe('Integration | Services | extractCertificationCandidatesFromCandidatesI
     });
   });
 
-  context('when billing information are provided', function () {
+  context('when billing feature toggle is enabled', function () {
     it('should return extracted and validated certification candidates with billing information', async function () {
```

Comment:
Ajouter context  `when certification center is PRO` + `when certification center is SCO`